### DISCUSSION TITLE
Safari 15 update

### DISCUSSION THREAD

**Discussion by luizcieslak** [mvasin/react-div-100vh#63] [General]
Hi All! This library is very helpful for full screen webapps, especially when used by Safari browser on mobile. I'm interested about what could change and how we will adapt these webapps after this upcoming safari update where the bottom tabs will be now a floating bar in the bottom. 

Here's some images and GIFs

![image](https://user-images.githubusercontent.com/14146176/121190704-b03ab900-c841-11eb-8b4d-0d0dfdefdf8d.png)

https://user-images.githubusercontent.com/14146176/121191281-3525d280-c842-11eb-87f1-1bd9037588be.mp4

https://twitter.com/evilrabbit_/status/1402054560584286209

![image](https://user-images.githubusercontent.com/14146176/121190883-d95b4980-c841-11eb-8f89-8e44e025dbe2.png)
https://twitter.com/stolinski/status/1402018905816698881/photo/1

**Comment by mvasin** [comment]
Hi @luizcieslak,

glad that you find the library useful. That was also my thinking when watching the presentation of iOS15. Unless I missed it, they only showed the outside-the-flow (in HTML sense) URL bar, like on the screenshot, for cases when a user already defined the URL, so it's not exactly clear how a user would type in the URL. It's nice that the window size doesn't change, but it also stands in the way, so the "native app-like" experience becomes questionable. And I'd be surprised if there would be an API to track which portion of the screen is covered, so the concern of covering calls to actions might be valid.

**Reply by luizcieslak** [reply]
Yeah, it was uncertain to me how that search bar is gonna behave after the URL is filled but I agree with you that this have some benefits tied to it. Currently I'm trying to be part of the beta and if I do I could share some interaction of that with this library.

**Reply by ivanjeremic** [reply]
Apple as always is to much putting its fingers in the HTML body where it has nothing to do and should stay away. The body is for the WebApp Developers and not for Apple to put stuff in there this is my opinion, however since the 100vh thing is fixed with this update will this library be still safe to use with it? I mean on older version it will fix the vh what about when it renders on this new updated version will it just work as expected?
